tests/acceptance/fixtures/responses/universal.js

```diff
@@ -0,0 +1,486 @@
+              latitude: 38.936519622802734,
+              longitude: -77.18428039550781
+            },
+            c_allstateLadyCTA: {
+              label: 'Learn More',
+              linkType: 'URL',
+              link: 'http://yext.com'
+            },
+            c_employeeCity: 'Tysons Corner',
+            c_employeeCountry: 'United States',
+            c_employeeDepartment: 'Consulting',
+            c_employeeRegion: 'Virginia',
+            c_employeeTitle: 'Associate Technical Project Manager',
+            c_oliverCta: {
+              url: 'https://www.yext.com/s/2287528/entity/edit2?entityIds=13411251',
+              icon: 'yext',
+              label: 'test cta'
+            },
+            c_startDate: '2017-09-18',
+            displayCoordinate: {
+              latitude: 38.924648,
+              longitude: -77.216859
+            },
+            emails: [
+              'afarooque@yext.com',
+              'af@yext.com',
+              'aplomb@yext.com'
+            ],
+            firstName: 'Amani',
+            geocodedCoordinate: {
+              latitude: 38.924654,
+              longitude: -77.216891
+            },
+            headshot: {
+              url: 'http://a.mktgcdn.com/p/bmc3W88h2mMRXk-wHQ7dB0Em4rR_dfia6OVrAK3LjYU/192x191.png',
+              width: 192,
+              height: 191,
+              sourceUrl: 'http://a.mktgcdn.com/p/bmc3W88h2mMRXk-wHQ7dB0Em4rR_dfia6OVrAK3LjYU/192x191.png'
+            },
+            languages: [
+              'German'
+            ],
+            lastName: 'Farooque',
+            mainPhone: '+18003332222',
+            routableCoordinate: {
+              latitude: 38.9242966,
+              longitude: -77.2177549
+            },
+            specialities: [
+              'Documentation',
+              'Coding'
+            ],
+            websiteUrl: {
+              url: 'http://www.test.com',
+              displayUrl: 'http://www.testdisplay.com',
+              preferDisplayUrl: true
+            },
+            yextDisplayCoordinate: {
+              latitude: 38.924648,
+              longitude: -77.216859
+            },
+            yextRoutableCoordinate: {
+              latitude: 38.9242966,
+              longitude: -77.2177549
+            }
+          },
+          uid: '18716057'
+        }
+      }
+    },
+    searchIntents: [],
+    locationBias: {
+      latitude: 38.8769,
+      longitude: -77.0893,
+      locationDisplayName: 'Arlington County, Virginia, United States',
+      accuracy: 'DEVICE'
+    }
+  }
+};
+const CORSHeaders = { 'access-control-allow-credentials': true, 'access-control-allow-origin': 'http://localhost:9999' };
+
+export const MockedUniversalSearchRequest = RequestMock()
+  .onRequestTo(async request => {
+    const urlRegex = /^https:\/\/liveapi.yext.com\/v2\/accounts\/me\/answers\/query\?input=amani\+farooque\+phone\+number/;
+    return urlRegex.test(request.url) && request.method === 'get';
```

Comment:
I believe you can set the method to only intercept GET request in RequestMock?

Reply:
Actually that might only be possible for RequestLogger https://github.com/yext/answers-hitchhiker-theme/blob/050beb6834e1c9fa2faac5291eb908ebce369395/tests/acceptance/searchrequestlogger.js#L60